cmd/chains.go

```diff
@@ -11,31 +11,32 @@ import (
 	"github.com/strangelove-ventures/lens/client/chain_registry"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-func chainsCmd() *cobra.Command {
+func chainsCmd(v *viper.Viper, lc *lensConfig) *cobra.Command {
```

Comment:
this gets put in after the config gets parsed right? (i.e. allows the CLI to keep working?)

Reply:
Yes, the lensConfig is populated here https://github.com/strangelove-ventures/lens/blob/f2f3a7c65c191d435ab36c9202f69e60975184c8/cmd/root.go#L56

and begins propagating down to the child commands later in the `NewRootCommand` function https://github.com/strangelove-ventures/lens/blob/f2f3a7c65c191d435ab36c9202f69e60975184c8/cmd/root.go#L85-L94